mobile/src/main/res/values-el/strings.xml

```diff
@@ -0,0 +1,108 @@
+    <string name="nfc_action_toggle">Μεταβολή</string>
+    <string name="nfc_dialog_title">Παρακαλώ επιλέξτε ενέργεια για την ετικέτα NFC</string>
+    <string name="settings_connection_title">Ρυθμίσεις Σύνδεσης</string>
+    <string name="settings_display_title">Ρυθμίσεις Προβολής</string>
+    <string name="settings_misc_title">Διάφορες Ρυθμίσεις</string>
+    <string name="settings_openhab_alturl">Απομακρυσμένη διεύθυνση openHAB</string>
+    <string name="settings_openhab_alturl_summary">Απομακρυσμένη διεύθυνση openHAB (χρησιμοποιείται όταν το τοπικό openHAB δεν είναι διαθέσιμο)</string>
+    <string name="settings_openhab_appversion">Έκδοση openHAB App</string>
+    <string name="settings_openhab_demomode">Κατάσταση επίδειξης</string>
+    <string name="settings_openhab_demomode_summary">Εκτέλεση σε κατάσταση επίδειξης</string>
+    <string name="settings_openhab_fullscreen">Πλήρης Οθόνη</string>
+    <string name="settings_openhab_fullscreen_summary">Προβολή σε πλήρη οθόνη</string>
+    <string name="settings_openhab_icon_format">Μορφή εικονιδίου</string>
+    <string name="settings_openhab_none">κανένα</string>
+    <string name="settings_openhab_password">Κωδικός</string>
+    <string name="settings_openhab_password_summary">Κωδικός openHAB</string>
+    <string name="settings_openhab_screentimeroff">Απενεργοποίηση αδρανοποίησης</string>
+    <string name="settings_openhab_screentimeroff_summary">Απενεργοποίηση σβησίματος οθόνης όταν τρέχει το openHAB</string>
+    <string name="settings_openhab_sslcert">Αγνόηση πιστοποιητικού SSL</string>
+    <string name="settings_openhab_sslcert_summary">Αγνόηση της επικύρωσης του πιστοποιητικού κατά την διαδικασία SSL Handshake</string>
+    <string name="settings_openhab_sslclientcert">Πιστοποιητικό SSL πελάτη</string>
+    <string name="settings_openhab_sslclientcert_howto">Πιστοποιητικό SSL πελάτη πώς-να</string>
+    <string name="settings_openhab_sslhost">Αγνόηση ονόματος διακομιστή SSL</string>
+    <string name="settings_openhab_sslhost_summary">Αγνόηση του ελέγχου ονόματος διακομιστή κατά την διαδικασία SSL Handshake</string>
+    <string name="settings_openhab_theme">Θέμα openHAB</string>
+    <string name="settings_openhab_url">Διεύθυνση openHAB</string>
+    <string name="settings_openhab_url_summary">Ρύθμιση διεύθυνσης openHAB με το χέρι. Απενεργοποιεί την αυτόματη εύρεση αν έχει ήδη ρυθμιστεί.</string>
+    <string name="settings_openhab_username">Όνομα χρήστη</string>
+    <string name="settings_openhab_username_summary">Το όνομα χρήστη (username) του openHAB</string>
+    <string name="settings_ringtone">Ήχος κλήσης</string>
+    <string name="theme_name_black">Μαύρο/AMOLED</string>
+    <string name="theme_name_dark">Σκούρο</string>
+    <string name="theme_name_light">Ανοιχτόχρωμο</string>
+    <string name="theme_value_black">Μαύρο</string>
+    <string name="theme_value_dark">Μαύρο</string>
+    <string name="theme_value_light">Ανοιχτόχρωμο</string>
```

Comment:
Please remove the three "theme_value_..." strings. They will become obsolete when #350 is merged